hack/jenkins/linux_integration_tests_virtualbox.sh

```diff
@@ -28,9 +28,10 @@ set -e
 OS_ARCH="linux-amd64"
 VM_DRIVER="virtualbox"
 JOB_NAME="VirtualBox_Linux"
+EXTRA_TEST_ARGS="-timeout-multiplier=2"
```

Comment:
the intent of the multiplier was for slow machines, I dont consider our GPU machines to be slow one. I don think this would help in fixing the failures

Reply:
virtualbox is just very slow in general, it doesn't really have anything to do with the machine in this case

Reply:
if we prove virtualbox is slow, we should fix that in minikube code not hack our way in tests.
in my experience vbox is slow to create machines, because it allocates memory for real. not dymanmically but after that it is solid and fast.  so in minikube code we could adjust the waiting for host creation for vbox to be more than other drivers.

this was meant to be used only in slow machines.

Reply:
removed